flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -702,11 +714,14 @@ private void disposeAllOperators(boolean logOnlyErrors) throws Exception {
 						operator.dispose();
 					}
 					catch (Exception e) {
```

Comment:
This is a bit inconsistent with other changes where we catch `Throwable`, not `Exception`.

Reply:
Hmm... Yes a little bit, is it safe to make the change? I think so.

Reply:
I think it's not out-of-scope because this PR changes error handling.

Reply:
safe to change I think, I will do it.